src/mscorlib/src/System/Runtime/Serialization/SerializationInfo.cs

```diff
@@ -408,16 +408,20 @@ internal void AddValueInternal(String name, Object value, Type type)
         /*=================================UpdateValue==================================
         **Action: Finds the value if it exists in the current data.  If it does, we replace
         **        the values, if not, we append it to the end.  This is useful to the 
-        **        ObjectManager when it's performing fixups, but shouldn't be used by 
-        **        clients.  Exposing out this functionality would allow children to overwrite
-        **        their parent's values.
+        **        ObjectManager when it's performing fixups
         **Returns: void
         **Arguments: name  -- the name of the data to be updated.
         **           value -- the new value.
         **           type  -- the type of the data being added.
-        **Exceptions: None.  All error checking is done with asserts.
+        **Exceptions: None.  All error checking is done with asserts. Although public,
+        **            it's not exposed in a contract and is only meant to be used by corefx.
+        **
+        **Visibility:
+        **This should not be used by clients: exposing out this functionality would allow children
+        **to overwrite their parent's values. It is public in order to give corefx access to it for
+        **its ObjectManager implementation, but it should not be exposed out of a contract.
         ==============================================================================*/
-        internal void UpdateValue(String name, Object value, Type type)
+        public void UpdateValue(String name, Object value, Type type)
```

Comment:
Ok.  Both model.xml and mscorlib.cs are specific to CoreCLR already such that there's no ifdef'ing needed there, right?